lib/compat/wordpress-6.6/resolve-patterns.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+if ( ! function_exists( 'gutenberg_replace_pattern_blocks' ) ) {
```

Comment:
We can remove the condition here. The callbacks in the plugin should use `gutenberg_` prefix and that should be enough to avoid name collisions.

Reply:
Yeah I think we have an exclusion rule for Gutenberg prefixes and suffixes I think.